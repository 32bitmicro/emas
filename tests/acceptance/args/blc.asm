blc 0x0000
blc 0x0100
blc 0xff00
