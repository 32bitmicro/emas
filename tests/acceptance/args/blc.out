@ 0x0000 : 0xe400  /  111 001 0 000 000 000  /  58368
@ 0x0001 : 0xe401  /  111 001 0 000 000 001  /  58369
@ 0x0002 : 0xe4ff  /  111 001 0 011 111 111  /  58623
